lib/txgh/handlers/github/delete_handler.rb

```diff
@@ -0,0 +1,71 @@
+require 'logger'
+
+module Txgh
+  module Handlers
+    module Github
+      class DeleteHandler
+        include ResponseHelpers
+
+        attr_reader :project, :repo, :payload, :logger
```

Comment:
seems like each of the handlers have these attributes in common, might be good to create a parent `Handler` class and then subclass it with each handler